### PR TITLE
work on issue #237

### PR DESCRIPTION
**Pull request by ktarbet**
Summary:

The code for copying Text records was using deprecated Fortran that didn't use the structure for text (zStructText).
During work I found ztextRetrieve didn't check the version of DSS Files.

Proposed Fix:
Add Text record specific logic to zcopyRecord , such as exists for paired-data, etc.
in ztextRetrieve handle DSS 6 files with a new method ztextRetrieve6.

### CODE REVIEW

**Review by perrymanmd — APPROVED** [review]

File: heclib/heclib_c/src/System/getCurrentTimeMillis.c [hunk 1]
```diff
@@ -32,6 +33,14 @@ long long getCurrentTimeMillis()
 {
 	long long ltime;
 
+	char* millis = getenv("DSS_CURRENT_TIME_MILLIS");
```
Comment (MikeNeilson) [inline heclib/heclib_c/src/System/getCurrentTimeMillis.c:36]: Unless you're going to change the environment in the middle of the run I would cache the actual time as a static, with a bool to indicate it was provided. I don't think getenv is a cheap call and that's probably going to be a lot of atoi calls too.

Comment (MikeNeilson) [reply heclib/heclib_c/src/System/getCurrentTimeMillis.c:36]: Granted, since performance, should probably be measured.

**Review by MikeNeilson — APPROVED** [review]
I have one possible nit pick, but it's a possible performance gotcha.

File: heclib/heclib_c/src/System/getCurrentTimeMillis.c (hunk 1 above)
Comment (ktarbet) [reply heclib/heclib_c/src/System/getCurrentTimeMillis.c]: good ideas -  I've updated the code.

**Pull request merged**